package com.growth.task.todo.exception;

public class NotFoundException extends RuntimeException {
    public NotFoundException() {
        super("Bad");
    }
}
